src/scripts/api.js

```diff
@@ -65,7 +65,7 @@ class SlawAPI{
       })
     ;
 
-    let channel = apiSocket.channel(connectionString);
+    let channel = apiSocket.channel(connectionString, parameters || {});
```

Comment:
Is this basically the same as changing the connection string to user_data:$v:$username?
If so we should move this one in the string in SlawAPI::config(), or move $username out of the gamble status one, for consistency.